routes/wow.py

```
@@ -90,4 +90,57 @@
+        ).json()
+
+        print(response)
+
+        if "data" not in response:
+            print(
+                f"Error no matching data with given inputs {json_data} response {response}"
+            )
+            if NO_RATE_LIMIT:
+                return f"Error no matching data with given inputs {json_data} response {response}"
+            # send generic error message to remove XSS potential
+            return f"error no matching results found matching search inputs"
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)